src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -301,6 +317,83 @@ undistribute_table(PG_FUNCTION_ARGS)
 }
 
 
+/*
+ * alter_distributed_table gets a distributed table and some other
+ * parameters and alters some properties of the table according to
+ * the parameters.
+ */
+Datum
+alter_distributed_table(PG_FUNCTION_ARGS)
```

Comment:
It would be kind of nice to move this logic into a new file, since this file is getting kind of huge.

Reply:
That makes sense, especially now we add things much less related to create_distributed_table.